internal/tasks/fleet_selector_test.go

```diff
@@ -106,14 +106,29 @@ var _ = Describe("FleetSelector", func() {
 			Expect(device.Metadata.Owner).To(BeNil())
 		})
 
-		It("Fleet selector removed no overlap", func() {
+		It("Nil fleet selector matches all", func() {
```

Comment:
We should make those 
```suggestion
		It("should match all for nil fleet selector", func() {
```

Reply:
Why?  This description is more difficult for me to understand.
How about "Nil fleet selector should match all devices"?

Reply:
Nice reference, now I understand what you meant.  I'll do an overhaul in a separate PR.